apps/backend/src/modules/git-emails/git-email.repository.ts

```diff
@@ -0,0 +1,58 @@
+import { type Repository } from "~/libs/types/types.js";
+
+import { GitEmailEntity } from "./git-email.entity.js";
+import { type GitEmailModel } from "./git-email.model.js";
+
+class GitEmailRepository implements Repository {
+	private gitEmailModel: typeof GitEmailModel;
+
+	public constructor(gitEmailModel: typeof GitEmailModel) {
+		this.gitEmailModel = gitEmailModel;
+	}
+
+	public async create(entity: GitEmailEntity): Promise<GitEmailEntity> {
+		const { contributor, email } = entity.toNewObject();
+
+		const gitEmail = await this.gitEmailModel
+			.query()
+			.insert({
+				contributor,
```

Comment:
I made this change to avoid redundant data (contributor and contributorId) in the entity (previous comment), and I pass in the contributor in the insert method to add contributor's id and name values to git-emails, because without it it sends `underfined`

Reply:
I didn't quite get what's the issue 😅 As I see you create contributor inside the log-activity service separately, and only after that you create a git_email, so the contributor already exists and we only need contributor_id. It's ok if we have both contributor and contributorId inside the entity, it sounds better to me to have it like this instead of passing unnecessary entity to the query